Documentation/architecture/net-platform-standard.md

```diff
@@ -1,5 +1,7 @@
 ﻿# .NET Platform Standard
 
+> The .NET Platform Standard concept has been replaced by the .NET Standard Library. Check out the [.NET Standard Library](https://docs.microsoft.com/dotnet/articles/standard/library) topic in the [.NET Documentation](https://docs.microsoft.com/dotnet). It provides a good first view into .NET Standard Library.
```

Comment:
I'm under the impression they are the same thing its just a name change to reduce concept count/confusion.  @richlander is that correct?  I've never understood what distinction we were trying to make with this term, see https://github.com/dotnet/corefx/pull/5729#discussion_r50991776.
I don't think it's right to say "replaced".  I would just change this doc to say its about .NET Standard library, remove any overlapping content, and point users to the main docs as "first reading".